providers-sdk/v1/inventory/deprecated_v8.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package inventory
+
+import "strings"
+
+func DeprecatedV8CompatAssets(assets []*Asset) {
+	for i := range assets {
+		DeprecatedV8CompatAsset(assets[i])
+	}
+}
+
+func DeprecatedV8CompatAsset(asset *Asset) {
+	if asset == nil {
+		return
+	}
+
+	if asset.Platform != nil {
+		asset.Platform.DeprecatedV8Kind = Kind2DeprecatedV8Kind(asset.Platform.Kind)
+	}
+
+	for i := range asset.Connections {
+		conn := asset.Connections[i]
+		conn.Kind = Kind2DeprecatedV8Kind(asset.KindString)
+	}
+
+	// FIXME: Remove this and solve it at its core
+	var ids []string
+	for _, id := range asset.PlatformIds {
+		if strings.HasPrefix(id, "//") {
```

Comment:
I noticed a bunch of platform IDs that shouldn't be sent up:

![image](https://github.com/mondoohq/cnquery/assets/1307529/e4a7320b-1892-4ee2-b241-a4324103124c)

Reply:
we need to fix these at their core, this is a sanity-check for v8, but it needs to be removed long-term